packages/workers-shared/asset-worker/src/index.ts

```diff
@@ -129,10 +125,6 @@ export default class extends WorkerEntrypoint<Env> {
 			},
 			this.unstable_exists.bind(this)
 		);
-		// if asset exists but non GET/HEAD method, 405
```

Comment:
We might still want this check in router worker, since we would rather return the user worker instead of a 405. I'm totally in agreance this makes sense as just a bool response!

Reply:
Can't change that behavior now without a compat flag.